buffer/folder.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  create_folders = var.enable && var.parent_folder_id != ""
+}
+
+resource "google_folder" "buffer_project_folder" {
```

Comment:
mind improving the name of this and maybe adding a comment? `buffer`, `project`, and `folder` are all fairly generic in this context, even if they're altogether.

Reply:
WDYT? It is local to this file and having a hard time renaming.